### PR TITLE
feat : contact us page & about us page

### LINKED ISSUE #5: About us page

### LINKED ISSUE #6: Contact us page

### PR DESCRIPTION
**Pull request by PeerapatPhetpadriew**
closes #5 
closes #6

### CODE REVIEW

**Comment by ppetpadriew** [comment]
file naming convention is `snake_case`. Use underscore to separate words.

**Comment by PeerapatPhetpadriew** [comment]
File name changed krub.

File: assets/css/main.css [hunk 1]
```diff
@@ -94,10 +93,85 @@ body {
...
+    height: 100%;
+    width: 45%;
+    background-color: green;
+    float: left;
+    padding-right: 5%;
+    border-right: 3px dashed gold;
+    text-align: center;
+}
+
+.blank-container {
```
Comment (ppetpadriew) [inline assets/css/main.css]: you should name it `map-container` because they will put the map at that blank space

File: contact_us.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			<div class="location__head">LOCATION</div>
+			<div class="location__sub">APH INDUSTRIAL GROUP</div>
+		</div>
+		<div class="blank-container">123</div>
+	</div>
+
+</div>
+
+<?php
+require_once __DIR__ . '/includes/footer.php';
```
Comment (ppetpadriew) [inline contact_us.php]: Don't forget to add new line at the end. You may consider this package https://stackoverflow.com/questions/37794699/sublime-text-3-ensure-only-one-trailing-newline-at-end-of-file which will automatically handle this for you.

Comment (ppetpadriew) [inline contact_us.php]: Don't forget to add new line at the end. You may consider this package https://stackoverflow.com/questions/37794699/sublime-text-3-ensure-only-one-trailing-newline-at-end-of-file which will automatically handle this for you.

**Comment by ppetpadriew** [comment]
Content is overflow on x-axis and result in scroll bar. Please fix this
image

**Pull request closed without merging**